checks/pinned_dependencies_test.go

```diff
@@ -542,3 +543,78 @@ func TestGitHubWorflowRunDownload(t *testing.T) {
 		})
 	}
 }
+
+func TestGitHubWorkflowUsesLineNumber(t *testing.T) {
```

Comment:
is there a way we could re-use the existing `TestGithubWorkflowPinning` instead of creating a new `TestGitHubWorkflowUsesLineNumber `?

Reply:
I personally think it's better having it in its own test because I prefer unit tests to focus on just one thing as opposed to several disparate things. Combining it with `TestGithubWorkflowPinning` would make it a bit too convoluted.

Reply:
I agree with @chrismcgehee